src/video/windows/SDL_windowsevents.c

```diff
@@ -861,30 +861,34 @@ static bool HasDeviceID(Uint32 deviceID, const Uint32 *list, int count)
 }
 
 #if !defined(SDL_PLATFORM_XBOXONE) && !defined(SDL_PLATFORM_XBOXSERIES)
-static void GetDeviceName(HDEVINFO devinfo, const char *instance, char *name, size_t len)
+static void GetDeviceName(const char *devName, const char *instance, char *name, size_t len)
```

Comment:
You'll need to fall back to the original implementation for devices that don't have a HID product name.